### PR TITLE
modifies update matrix servlet to use body instead of parameters

### PR DESCRIPTION
**Pull request by AgusQuintanar**

### CODE REVIEW

**Review by Alexcgzz — APPROVED** [review]

**Review by Alexcgzz — APPROVED** [review]

File: src/main/java/com/google/sps/servlets/UpdateMatrixServlet.java [hunk 1]
```diff
@@ -9,31 +9,70 @@
...
+			Gson gson = new Gson();
+
+			String updatedStringMatrix = matrix.getStringMatrix();
 
-		InfectionProbability infectionProbability = new InfectionProbability(IP_NO_PROTECTION, IP_WITH_MASK, IP_WITH_VACCINE);
+			response.setContentType("application/json;");
+			response.getWriter().println(gson.toJson(updatedStringMatrix));
+		} 
+		catch (JSONException e) {
+		// crash and burn
```
Comment (mkayala-g) [inline src/main/java/com/google/sps/servlets/UpdateMatrixServlet.java:63]: Indent

File: src/main/java/com/google/sps/Matrix.java [hunk 2]
```diff
@@ -299,11 +293,12 @@ public String getStringMatrix() {
     
 
     public static void main(String[] args) {
-        InfectionProbability ip = new InfectionProbability(.855, .206, .53);
+        /*InfectionProbability ip = new InfectionProbability(.855, .206, .53);
         Matrix matrix = new Matrix("4,4,0000735000000000", ip);
         System.out.println(matrix.getStringMatrix());
         matrix.updateMatrix();
         System.out.println(matrix.getStringMatrix());
+        */
```
Comment (mkayala-g) [inline src/main/java/com/google/sps/Matrix.java]: Maybe completely remove this main function if no longer used?

**Review by mkayala-g — APPROVED** [review]
Hey - just introducing myself mkayala@sps-program.com (mkayala@google.com). I'll be review ing some PRs (and am available in your group chat!)

File: src/main/java/com/google/sps/servlets/UpdateMatrixServlet.java (hunk 1 above)
Comment (AgusQuintanar) [reply src/main/java/com/google/sps/servlets/UpdateMatrixServlet.java:63]: Hi, nice to meet you :), oh yes let me change that.

**Review by edgar-granados-perez — APPROVED** [review]

**Pull request merged**